src/Tests/Features/OfferMatching/ListTests.cs

```diff
@@ -64,7 +65,32 @@ public async Task GivenAnyAuthenticatedUser_WhenThereIsOnePlayerWhoIsInOfferMatc
         freeAgent.Position.Should().Be(mockPlayer.Position);
         freeAgent.HeadShotUrl.Should().Be(mockPlayer.HeadShotUrl);
         freeAgent.OfferingTeam.Should().Be(mockTeam.Name);
-        freeAgent.Offer.Should().Be(bidAmount.ToString("C0"));
+        freeAgent.Offer.Should().Be(bidAmount);
+    }
+    [Fact]
+    public async Task GivenAnyAuthenticatedUser_AllowPlayerToBeMatched()
+    {
+        var application = CreateUserAuthenticatedApplication();
+
+        var player = CreateFakePlayer();
+        player.Position = Position.QuarterBack.Name;
+        player.YearContractExpires = DateTime.Now.Year;
+        await application.AddAsync(player);
+        var request = CreateFakeValidRequest();
```

Comment:
if this is used in one spot it might be better to just have the autofaker do it in here.